browser/android/youtube_script_injector/java_sources.gni

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+brave_youtube_script_injector_java_sources = [ "../../brave/browser/android/youtube_script_injector/java/org/chromium/chrome/browser/BraveYouTubeScriptInjectorNativeHelper.java" ]
```

Comment:
see above, do not use java_sources.gni for brave-specific code. It should only be used when overriding upstream code that lives in `chrome/android/java`